src/mds/MDCache.cc

```diff
@@ -11520,6 +11525,9 @@ void MDCache::force_readonly()
 
 void MDCache::show_subtrees(int dbl)
 {
+  if (g_conf->mds_thrash_exports)
+    dbl += 5;
```

Comment:
Is this bit supposed to be here? If you want it in-tree please make it a new commit.